packages/wrangler/src/__tests__/secret.test.ts

```diff
@@ -226,22 +267,30 @@ describe("wrangler secret", () => {
 				expect(std.err).toMatchInlineSnapshot(`
 			"[31mX [41;31m[[41;97mERROR[41;31m][0m [1m🚨 Error uploading secret for key: secret-name-1:[0m
 
-			                  Failed to create secret 1
+			                  request to
+			  [4mhttps://api.cloudflare.com/client/v4/accounts/some-account-id/workers/scripts/script-name/secrets[0m
```

Comment:
wait, is this what the user sees when this endpoint fails?

Reply:
When there is a network error, yeah I believe so. Previously the tests were throwing a General Error into the Jest test lol

Reply:
https://github.com/cloudflare/wrangler2/pull/2425#discussion_r1053633920